<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ColorPlayer</string>
  </property>
  <widget class="VideoWidget" name="openGLWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>461</width>
     <height>271</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
  </widget>
  <widget class="XSlider" name="PlaySlider">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>320</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="pageStep">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="PlayTime">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="TotalTime">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>320</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="XSlider" name="AudioSlider">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>320</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="pageStep">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>videowidget.h</header>
  </customwidget>
  <customwidget>
   <class>XSlider</class>
   <extends>QSlider</extends>
   <header>xslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>AudioSlider</sender>
   <signal>sliderPressed()</signal>
   <receiver>Widget</receiver>
   <slot>AudioSlider_pressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AudioSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>Widget</receiver>
   <slot>AudioSlider_released()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AudioSlider_pressed()</slot>
  <slot>AudioSlider_released()</slot>
 </slots>
</ui>
